ui/widgets/datepicker.js

```diff
@@ -172,6 +176,9 @@ var widget = $.widget( "ui.datepicker", {
 		},
 		blur: function() {
 			this.suppressExpandOnFocus = false;
+		},
+		change: function( event ) {
```

Comment:
The native `change` event should be fine for user input. It's only programmatic value changes that would cause the event to not be fired.